examples/dis7/Connection.cpp

```diff
@@ -0,0 +1,99 @@
+#include "Connection.h"
```

Comment:
This is common for both Examples, so it would be ideal to just have the one copy. could go in an examples common folder, with Logging & Timer classes.

Reply:
Thats a great idea. I was also thinking about splitting the utils directory to common as well to break up the dis6 and dis7 dependencies.

Reply:
moved things into a common directory.